app/helpers/sac_cas/event/participation_banner.rb

```diff
@@ -30,4 +30,18 @@ def render_cancel_popover
     @context.render('event/participations/popover_participant_cancels',
                     entry: @user_participation).to_s
   end
+
+  def status_text
+    t(@user_participation.state, scope: 'event.participations.states')
+  end
+
+  def status_class
+    alert_class = {
+      assigned: :success, attended: :success, summoned: :success,
+      unconfirmed: :warning, applied: :warning, absent: :warning,
+      annulled: :danger, canceled: :danger, rejected: :danger
+    }[@user_participation.state.to_sym]
```

Comment:
@mtnstar Should we handle the case when `alert_class` may be `nil`? For example if a new state gets added? We could either `raise` or log an error for example.

Reply:
no, I think it's fine like this